__tests__/integration-webhooks.test.js

```diff
@@ -82,8 +82,22 @@ describe('Metrics SDK Webhook Integration Tests', () => {
     return httpServer.kill();
   });
 
-  it('should return with a user object if the signature is correct', async () => {
-    const time = Date.now();
+  it('should return with a 401 if the signature is empty/missing', async () => {
+    const response = await post(`http://localhost:${PORT}/webhook`, JSON.stringify({ email: 'dom@readme.io' }), {
+      headers: {
+        'content-type': 'application/json',
+      },
+    });
+    expect(response.statusCode).toBe(401);
+
+    const responseBody = await getResponseBody(response);
+    expect(responseBody.error).toBe('Missing Signature');
+  });
+
+  it('should return an error with an expired signature', async () => {
+    // Expiry time is 30 mins
```

Comment:
```suggestion
    // Expiry time is 40 mins
```

Reply:
See this f6ab3a5d3aefdafc5cdcf3786b32fce43568176d made the comment a little clearer